tests/backup/backup_restore_basic_test.go

```diff
@@ -1318,3 +1319,310 @@ var _ = Describe("{BackupSyncBasicTest}", func() {
+			scheduleNames = append(scheduleNames, scheduleBkpSingleNs)
+		})
+		Step("Creating a schedule backup for multiple applications with namespace label filter", func() {
+			log.InfoD("Creating a schedule backup for multiple applications with namespace label filter")
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Unable to fetch px-central-admin ctx")
+			scheduleBkpMultipleNs = fmt.Sprintf("%s-%v", BackupNamePrefix, time.Now().Unix())
+			err = CreateScheduleBackupWithNamespaceLabel(scheduleBkpMultipleNs, SourceClusterName, backupLocationName, backupLocationUID,
+				labelSelectors, orgID, "", "", "", "", labelForMultipleNamespace, periodicSchedulePolicyName, periodicSchedulePolicyUid, ctx)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying creation of schedule backup [%s] with labels [%v]", scheduleBkpMultipleNs, labelForMultipleNamespace))
+			firstScheduleBackupForMultipleNs, err = GetFirstScheduleBackupName(ctx, scheduleBkpMultipleNs, orgID)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Fetching the name of the first schedule backup [%s]", firstScheduleBackupForMultipleNs))
+			err = NamespaceLabelBackupSuccessCheck(firstScheduleBackupForMultipleNs, ctx, multipleNamespace, labelForMultipleNamespace)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying if the labeled namespaces [%v] are backed up and check for labels [%s] applied to backups [%s]", multipleNamespace, labelForMultipleNamespace, firstScheduleBackupForMultipleNs))
+			scheduleNames = append(scheduleNames, scheduleBkpMultipleNs)
+		})
+		Step("Creating a schedule backup for all applications with namespace label filter", func() {
+			log.InfoD("Creating a schedule backup for all applications with namespace label filter")
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Unable to fetch px-central-admin ctx")
+			scheduleBkpAllNs = fmt.Sprintf("%s-%v", BackupNamePrefix, time.Now().Unix())
+			err = CreateScheduleBackupWithNamespaceLabel(scheduleBkpAllNs, SourceClusterName, backupLocationName, backupLocationUID,
+				labelSelectors, orgID, "", "", "", "", labelForAllNamespace, periodicSchedulePolicyName, periodicSchedulePolicyUid, ctx)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying creation of schedule backup [%s] with labels [%v]", scheduleBkpAllNs, labelForAllNamespace))
+			firstScheduleBackupForAllNs, err = GetFirstScheduleBackupName(ctx, scheduleBkpAllNs, orgID)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Fetching the name of the first schedule backup [%s]", firstScheduleBackupForAllNs))
+			err = NamespaceLabelBackupSuccessCheck(firstScheduleBackupForAllNs, ctx, bkpNamespaces, labelForAllNamespace)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying if the labeled namespaces [%v] are backed up and check for labels [%s] applied to backups [%s]", bkpNamespaces, labelForAllNamespace, firstScheduleBackupForAllNs))
+			scheduleNames = append(scheduleNames, scheduleBkpAllNs)
+		})
```

Comment:
Here also same you can write a common method.

Reply:
same